notebooks/Bulk Update Maps for Use in Collector or Field Maps.ipynb

```diff
@@ -0,0 +1,194 @@
+      "\n",
+      "\n",
+      "4 web maps found. Are you sure you want to work on all these web maps? (y/N): y\n"
+     ]
+    }
+   ],
+   "source": [
+    "webmap_search_query = webmap_search_query + f\" AND owner:{gis.users.me.username}\"\n",
+    "webmaps = gis.content.search(webmap_search_query, item_type='Web Map', max_items=1000, outside_org=False)\n",
+    "print(webmaps)\n",
+    "print(\"\\n\")\n",
+    "if len(webmaps) > 0:\n",
+    "        ylist = ['yes', 'y', 'ye']\n",
+    "        yn = (input(str(len(webmaps)) + ' web maps found. Are you sure you want to work on all these web maps? (y/N): ') or 'n').lower().strip()\n",
+    "        sure = yn.lower() in ylist\n",
+    "else:\n",
+    "    print (\"No items found with query:\", webmap_search_query)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### If you want to continue, we'll now update the keywords needed to use Collector and Field Maps"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Checking whether to update webmap for Field Maps: Hydrant Inspections\n",
+      "New keyword added for Hydrant Inspections. Updating webmap\n",
+      "\n",
+      "\n",
+      "Checking whether to update webmap for Field Maps: Hydrant Inspections_dispatchers\n",
+      "New keyword added for Hydrant Inspections_dispatchers. Updating webmap\n",
+      "\n",
+      "\n",
+      "Checking whether to update webmap for Field Maps: Hydrant Inspection\n",
+      "New keyword added for Hydrant Inspection. Updating webmap\n",
+      "\n",
+      "\n",
+      "Checking whether to update webmap for Field Maps: Hydrant Inspections_workers\n",
+      "New keyword added for Hydrant Inspections_workers. Updating webmap\n",
+      "\n",
+      "\n",
+      "Done!\n"
+     ]
+    }
+   ],
+   "source": [
+    "if not sure:\n",
+    "    raise Exception(\"Script terminated. Please check your search string again\")\n",
+    "else:\n",
+    "    for wm in webmaps:\n",
+    "        print(\"Checking whether to update webmap for Field Maps and Collector: \" + wm.title)\n",
+    "        current_keywords = set(wm.typeKeywords)\n",
+    "        \n",
+    "        if field_maps_usage == \"enabled\":\n",
+    "            current_keywords.discard(\"FieldMapsDisabled\")\n",
+    "        elif field_maps_usage == \"disabled\":\n",
+    "            current_keywords.add(\"FieldMapsDisabled\")\n",
+    "\n",
+    "        if collector_usage == \"enabled\":\n",
+    "            current_keywords.discard(\"CollectorDisabled\")\n",
+    "            current_keywords.add(\"Collector\")\n",
+    "        elif collector_usage == \"disabled\":\n",
+    "            current_keywords.discard(\"Collector\")\n",
```

Comment:
Do you need to do this? I'm asking because currently, you could enable Collector for map that might not actually support it.

My understanding is that when saving in map, if the map could be used in Collector, then this typekeyword is added. If the map also has typekeyword `CollectorDisabled` then it will not appear in Collector (even though it could work).